dips/dip-1.mdx

```diff
@@ -11,141 +11,77 @@ import useBaseUrl from '@docusaurus/useBaseUrl';
-
-All transmitted requests/responses are signed by the sending party using the [JWS Signature standard](#details-of-jws-signature-scheme) (with the Ed25519 / EdDSA cipher suite, and `compact` encoding).  The party's on-chain compliance key shall be used to sign these messages. This ensures all information and meta-data about payments is authenticated and cannot be repudiated.
-
-
-## Basic Protocol CommandRequest / CommandResponse Interaction
-
-Assume two VASPs A and B.  The basic protocol interaction consists of:
-
-* An initiating VASP A creates a `CommandRequestObject` containing a Command of the desired type.  Commands inform the other VASP what to create or mutate. Every Command refers to one or more Objects to create or update.
-* VASP A packages the Command via JWS using EdDSA and compact encoding.
-* VASP A establishes a connection to VASP B and sends the packaged Command to VASP B in the body of an HTTP POST.
-* VASP B listens for requests, and when received, verifies VASP A's signature and then processes the request to generate and send `CommandResponseObject` responses, with a success or failure status, through the HTTP response body. In case of an error that prevents VASP B from parsing an incoming `CommandRequestObject` an HTTP error code is returned.
-* The initiating VASP A receives the response, verifies its signature, and processes it to assess whether it was successful or not.
-
-If VASP A fails to receive a response from VASP B, or in the case of specific protocol failure codes, it must resend the request at some cadence until a response is received to ensure consistency.
-
-<img alt="Command Exchange" src={useBaseUrl('img/command_exchange.png')} />
-
-As mentioned in [Terminology](#terminology), every Object type contains a `reference_id` field which is a unique reference ID for the Object. At each state of an Object, only one VASP is allowed to mutate it (or none if the Object is in a terminal state). If the other VASP tries to mutate the object, the command will be rejected. This prevents concurrent attempts to update Objects that could lead to inconsistencies.
-
+The HTTP response sender must use the compliance key of the VASP account linked with the responder's address to sign the response JWS body. The response receiver uses this address to find the appropriate compliance key to verify the signed request. For example: VASP A transfers funds to VASP B. The HTTP response from B to A contains `X-DIEM-ADDRESS` as VASP B's address.
```

Comment:
The request sender already knows a receiver diem address, should it expect same address here? Or different x-diem-address is ok? I will assume they are same and validate the x-diem-address is same (same parent vasp or dd account address)

Reply:
I didn't introduce this, who did introduce X-DIEM-ADDRESS? I am very much in favor of deleting it or making this a could / optional field.

Reply:
Discussed offline, there are a dozen different ways to handle this than introducing a new field, so I will strip this out.

Reply:
deleted and updated in the current update